deps/deps.go

```diff
@@ -189,6 +189,23 @@ var dependencies = map[string]Dependency{
 		PreInstalled: []string{"docker", "crictl"},
 	},
 	"hdiutil": {},
+	//the kubebuilder testenv binaries are all in the same tarball
+	//installing any one will result in all three being installed
+	"test-etcd": {
+		Version:      "1.19.2",
+		Linux:        "https://storage.googleapis.com/kubebuilder-tools/kubebuilder-tools-linux-amd64.tar.gz",
```

Comment:
I think these are missing the `{{.version}}`

Reply:
The binary names need to match what is inside the archive e.g. etcd and kube-api-server

Reply:
There is already an entry for kubectl in the dependecy map that will install the official kubectl, not the one in the kubebuilder tools install.  Do you know if they are the same?